src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -62,7 +62,15 @@ IConfiguration configuration
 
 
         // API change detection for PR will pull artifact from devops artifact
-        public async Task DetectApiChanges(string buildId, string artifactName, string originalFileName, string commitSha, string repoName, string packageName, int prNumber, string codeFileName = "")
+        public async Task DetectApiChanges(string buildId, 
+            string artifactName, 
+            string originalFileName, 
+            string commitSha, 
+            string repoName, 
+            string packageName, 
+            int prNumber,
+            string reviewUrlPrefix,
```

Comment:
Why do we need to pass this prefix now?

Reply:
We are using that URL to comment in PR. It was earlier hardcoded to production URL but this change is to pass based on the details current instance is running instead of hardcoding apiview URL.

Reply:
Do we care about the scheme? I'd assume we will always use https for the link. It might be simpler to just take in the hostName rather than the urlPrefix, and then create the entire url in one place.